properties.ini.sample

```diff
@@ -0,0 +1,146 @@
+; this file is used to define all necessary options for virt-who testing
+; all default values are null, need to update according to the test requirements
+; hypervisor-builder can help to get some hypervisor and guest information
+
+[virtwho]
+; host= is the rhel system to run virt-who service
+; repo= is the base url of the rhel compose
```

Comment:
If you want to add comments, please think about the following format, add all the comments to the above line of the section.

; The section of [virtwho] is used to define the host information where virt-who package is installed and running 
; host is a required option to indicate the ip address or hostname
; username and password are the required options to access the host 
; port is an optional settings, the default port is 22 
; repo is an optional settings, to indicate where is the virt-who yum repository
[virtwho]
host=
username=
password=